sdk/monitor/Azure.Monitor.OpenTelemetry.LiveMetrics/src/Internals/Filtering/Filter.cs

```diff
@@ -91,6 +91,46 @@ public Filter(FilterInfo filterInfo)
 
             this.info = filterInfo;
 
+            if (typeof(TTelemetry) == typeof(Request))
+            {
+                if (filterInfo.FieldName == "Success")
+                {
+                    filterInfo = new FilterInfo("Extension_IsSuccess", filterInfo.Predicate, filterInfo.Comparand);
+                }
```

Comment:
nit are there any tests for this?
I'm asking because I added "Extension_IsSuccess", it's not included in the generated models.
If I renamed this property, would we be able to catch this? (I'm not planning on renaming anything)

Alternatively, consider using `nameof(<PropertyName>)` for all the properties here. They'll be less likely to break in the future.

Reply:
Updated to use `nameof`.